docs/help-center/your-account/account-settings.mdx

```diff
@@ -489,6 +489,31 @@ Depending on the state of the chat backup, you have two options:
 
 In both cases, make sure to store the generated passphrase in a secure place.
 
+## Chat settings
+
+You can alter the press Enter action in this section.
```

Comment:
I think this will need an own H3 headline like "Manage action on enter key" or something like this since the settings category sounds like there could be more different settings in the future.

Reply:
Thank you. Added extra information.